src/navigation/Nav.jsx

```diff
@@ -140,6 +141,12 @@ export class Nav extends React.Component {
 		);
 
 		let unauthItems = [
+			media.isAtMediumUp && create && {
```

Comment:
Can a storybook story get added to show this in the nav?

Reply:
Also I've added case where we do not have create